modules/campaignion_email_protest/campaignion_email_protest.info

```diff
@@ -2,71 +2,6 @@ name = Campaignion Email Protest
 description = Definitions for Advocacy Engine email protest content type
```

Comment:
This would be a good opportunity to replace "Advocacy Engine" with "Campaignion" :)